### PR TITLE
Prevent overly long lines of text

### PR DESCRIPTION
**Pull request by axemonkey**
[Pivotal](https://www.pivotaltracker.com/story/show/122198119)

Wraps most text in a `.text` block to keep the line length down.

### CODE REVIEW

File: app/assets/javascripts/modules/form-name.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ window.moj.Modules.FormName = {
     var self = this;
 
     if (self.$identifier.val().length > 0 && self.$unknown.is(':checked')) {
-      self.$unknown.attr('checked', false);
```
Comment (novotnyjakub) [inline app/assets/javascripts/modules/form-name.js]: Is this wrongly generated in the backend?

Comment (axemonkey) [inline app/assets/javascripts/modules/form-name.js]: No, the problem was that when the box was unchecked programattically the label remained highlighted. 

This should have been in a separate PR though – it's nothing to do with line lengths... my bad.

Comment (axemonkey) [inline app/assets/javascripts/modules/form-name.js]: I'll take it out of this one, force push, and then make a new smaller PR for this change.

**Comment by novotnyjakub** [comment]
👍  looks good

**Pull request merged**